src/main/java/com/segment/analytics/android/integrations/comscore/ComScoreIntegration.java

```diff
@@ -125,11 +128,20 @@ private void setNullIfNotProvided(
 
     Map<String, String> asset = mapSpecialKeys(properties, mapper);
 
-    int contentAssetId = properties.getInt("assetId", 0);
+    String contentAssetId = properties.getString("assetId");
+      if (contentAssetId == null || contentAssetId.trim().isEmpty()) {
+        contentAssetId = properties.getString("asset_id");
+        if (contentAssetId == null || contentAssetId.trim().isEmpty()) {
+          contentAssetId = "0";
+        }
+      }
```

Comment:
Indent fix:

```suggestion
    if (contentAssetId == null || contentAssetId.trim().isEmpty()) {
      contentAssetId = properties.getString("asset_id");
      if (contentAssetId == null || contentAssetId.trim().isEmpty()) {
        contentAssetId = "0";
      }
    }
```

Reply:
thank you @mericsson, I committed this indenting fix. sorry about that!